.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "react-app"
+}
```

Comment:
Please ensure to add an extra empty line at the end of each files (as it is required by github), look at the final post of this thread https://github.com/Microsoft/vscode/issues/12076